cmp/grid/filter/GridFilterFieldSpec.ts

```diff
@@ -82,10 +82,17 @@ export class GridFilterFieldSpec extends BaseFilterFieldSpec {
         // Get values from current column filter
         const filterValues = [];
         columnFilters.forEach(filter => {
-            const newValues = castArray(filter.value).map(value => {
-                value = sourceField.parseVal(value);
-                return filterModel.toDisplayValue(value);
-            });
+            // The parseVal of tag will castArray the value a second time, so make sure to flatten at the end.
+            const newValues = flatten(
+                castArray(filter.value)
+                    .map(value => {
+                        value = sourceField.parseVal(value);
+                        return filterModel.toDisplayValue(value);
+                    })
+                    // Filter out the null tag value as it isn't a valid value to display in value lists.
+                    // It is set by 'is blank'/'is not blank' filters.
+                    .filter(it => isEqual(it, [null]))
```

Comment:
seems safe enough, independent of whether we continue to put `null` in as filter values.